src/dataset-rotation/README.md

```diff
@@ -31,4 +31,28 @@ Manages rotation mechanism of incoming datasets.
       }
     ]
 }
+```
+### Developement
+
+#### VS Code Debugger config 
+
+```json
+{
+    // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
+    // Zeigen Sie auf vorhandene Attribute, um die zugehörigen Beschreibungen anzuzeigen.
+    // Weitere Informationen finden Sie unter https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Comments are not allowed in JSON and being strictly this is no valid JSON. Maybe we better remove the comment. Also it is in German...